playwright/components/index.ts

```diff
@@ -25,6 +26,10 @@ export const commonDataElementInputPreview = (page: Page) => {
   return page.locator(COMMMON_DATA_ELEMENT_INPUT);
 };
 
+export const iconButton = (page: Page) => {
```

Comment:
**suggestion**: this can just be called `button` instead of `iconButton` as the locator is just finding `type="button"` and has no overlap with the `IconButton` component, i'd rename this here at least 👍